bin/cliain/src/keys.rs

```diff
@@ -29,3 +31,14 @@ pub fn rotate_keys<C: AnyConnection>(connection: C) {
     let new_keys = rotate_keys_raw_result(&connection).expect("Failed to retrieve keys");
     info!("Rotated keys: {:?}", new_keys);
 }
+
+pub fn next_session_keys(connection: &Connection, account_id: String) {
+    let account_id = AccountId::from_ss58check(&account_id).expect("Address is valid");
+    match get_next_session_keys(connection, account_id) {
+        Some(keys) => {
+            println!("{}", keys.aura.encode_hex::<String>());
```

Comment:
Well, that was my first idea, but maybe we do not want that - this format seems to be very specific to this use case.
And concatenating two strings in some external script does not seem like to much work.

Reply:
Okay make sense, but can we then print them so they won't be swapped by mistake, either of below option is fine for me
```
aura: 0x...
aleph: 0x...
```
or as json:
```
{ 
   "aura": "0x..."
   "aleph": "0x..."
}
```
Preferably I like json, it can be parsed than by other tools and by simple text parsing `cliain [...] | grep "aura"`

Reply:
Can we please use `json!` macro, e.g.
```
fn json_req(method: &str, params: Value, id: u32) -> Value {
    json!({
        "method": method,
        "params": params,
        "jsonrpc": "2.0",
        "id": id.to_string(),
    })
}
```
see example here https://github.com/Cardinal-Cryptography/aleph-node/blob/main/aleph-client/src/rpc.rs#L13